tasks/windows_resources.py

```diff
@@ -47,12 +47,8 @@ def build_rc(ctx, rc_file, vars=None, out=None):
     ctx.run(command)
 
 
-def versioninfo_vars(
-    ctx,
-    major_version='7',
-    python_runtimes='3',
-):
-    py_runtime_var = get_win_py_runtime_var(python_runtimes)
+def versioninfo_vars(ctx, major_version='7'):
+    py_runtime_var = get_win_py_runtime_var()
```

Comment:
Should we inline `get_win_py_runtime_var`?

Reply:
Indeed, done